Documentation/ApiOverview/Environment/Index.rst

```diff
@@ -79,7 +79,7 @@ getConfigPath()
 ---------------
 
 The environment provides the path to :file:`typo3conf`. This folder contains TYPO3
-global configuration files and folders, e.g. :file:`LocalConfiguration.php`.
+global configuration files and folders, e.g. :file:`config/system/settings.php`.
```

Comment:
"The environment provides the path to typo3conf."

This should be rephrased.

Reply:
Have you looked it up in the Code? I haven't seen a ChangeLog entry, that it changed now?

Reply:
Ok, seems like this changed too

Reply:
I would suggest we make a follow up where we take care of all remains of "typo3conf" and the Enviroment API changes?

Reply:
Go ahead and please create a ticket, so it does not gets lost.

Reply:
done